tests/CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
```

Comment:
The last I checked, Ubuntu 18.04 only supports cmake version 3.10.  Since that's probably decent % of Linux dev out there, what do we complicate if we limit ourselves to 3.10 features?

Reply:
freertos and it's libs (coremqtt etc) require 3.13. So we are essentially staying up to date with those